src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -52,12 +50,10 @@ public RobotContainer() {
    */
   private void configureBindings() {
     driveTrain.setDefaultCommand(new DriveCommand(driveTrain, joystick::getY, joystick::getX));
-    // Just for testing
-    new JoystickButton(joystick, 1).toggleOnTrue(new BalancePID(driveTrain));
```

Comment:
We still need a way to enable the "balance mode" manually via the joytstick for the end game.